.github/workflows/job-precommit-python-dataframe.yml

```diff
@@ -0,0 +1,76 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Oh, do we need to remove the corresponding Jenkins file?

Reply:
Hi Danny!
If I'm not mistaken this one didn't have a jenkins file.